ndvi_forecast/2_train.ipynb

```diff
@@ -14,7 +14,7 @@
       "metadata": {},
       "source": [
         "# Train NDVI Forecast Model\n",
-        "In this notebook, EVI forecast model is built using satellite and weather (historical and forecast) data on dataset (curated locations spread across USA from Crop Data Layer), which predicts NDVI for next 10 days in advance for an Area of Interest (AOI). This notebook runs on the data downloaded from 1_download_data.ipynb notebook. "
+        "In this notebook, EVI forecast model is built using satellite and weather (historical and forecast) data on dataset (curated locations spread across USA from Crop Data Layer), which predicts NDVI for next 10 days in advance for an Area of Interest (AOI). This notebook runs on the data downloaded from [1_download_data.ipynb](./1_download_data.ipynb) notebook. "
```

Comment:
Title says NDVI forecast model. Test says EVI.

Reply:
Changed to NDVI